src/Cli/dotnet/Commands/Test/TestApplication.cs

```diff
@@ -115,6 +115,16 @@ private string GetArguments(TestOptions testOptions)
             builder.Append($" {TestingPlatformOptions.ResultsDirectoryOption.Name} {ArgumentEscaper.EscapeSingleArg(resultsDirectoryPath)}");
         }
 
+        if (_buildOptions.PathOptions.ConfigFilePath is { } configFilePath)
+        {
+            builder.Append($" {TestingPlatformOptions.ConfigFileOption.Name} {ArgumentEscaper.EscapeSingleArg(configFilePath)}");
```

Comment:
are we guaranteed that all runners using latest `dotnet test` understand these command line arguments?

Reply:
Yes. These command-line options existed before the MTP protocol I think. We also are only forwarding them if the user originally specified them.